hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/util/CryptoTransferHelper.java

```diff
@@ -0,0 +1,131 @@
+     * @return TokenTransferList containing the NFT transfer
+     */
+    public static TokenTransferList createNftTransfer(final TokenID tokenId, final NftTransfer nftTransfer) {
+        return TokenTransferList.newBuilder()
+                .token(tokenId)
+                .nftTransfers(nftTransfer)
+                .build();
+    }
+
+    /**
+     * Creates a {@link TokenTransferList} for a non-fungible token (NFT) transfer.
+     *
+     * @param tokenId the TokenID of the NFT to be transferred
+     * @param nftTransfers the list of NftTransfer objects
+     * @return TokenTransferList containing the NFT transfer
+     */
+    public static TokenTransferList createNftTransfer(final TokenID tokenId, final List<NftTransfer> nftTransfers) {
+        return TokenTransferList.newBuilder()
+                .token(tokenId)
+                .nftTransfers(nftTransfers)
+                .build();
+    }
+
+    /**
+     * Constructs a {@link NftTransfer} object for transferring a non-fungible token.
+     *
+     * @param from the sender's account ID
+     * @param to the receiver's account ID
+     * @param serialNo the serial number of the NFT
+     * @return NftTransfer object detailing the sender, receiver, and NFT serial number
+     */
+    public static NftTransfer nftTransfer(
+            @NonNull final AccountID from, @NonNull final AccountID to, final long serialNo) {
+        return NftTransfer.newBuilder()
+                .serialNumber(serialNo)
+                .senderAccountID(from)
+                .receiverAccountID(to)
+                .build();
+    }
+
+    /**
+     * Creates an {@link AccountAmount} representing a debit from an account.
+     *
+     * @param account the account from which funds will be debited
+     * @param amount  the amount to debit
+     * @return AccountAmount specifying the account and the negative amount debited
+     */
+    public static AccountAmount debit(@NonNull final AccountID account, final long amount) {
+        return adjust(account, -amount);
+    }
+
+    /**
+     * Creates an {@link AccountAmount} representing a credit to an account.
+     *
+     * @param account the account to which funds will be credited
+     * @param amount  the amount to credit
+     * @return AccountAmount specifying the account and the positive amount credited
+     */
+    public static AccountAmount credit(@NonNull final AccountID account, final long amount) {
+        return adjust(account, amount);
+    }
+
+    /**
+     * Helper method to create an {@link AccountAmount} with a specified adjustment.
+     *
+     * @param account the account to be adjusted
+     * @param amount  the amount of the adjustment (can be positive or negative)
+     * @return AccountAmount with the specified adjustment
+     */
+    private static AccountAmount adjust(@NonNull final AccountID account, final long amount) {
+        return AccountAmount.newBuilder().accountID(account).amount(amount).build();
```

Comment:
Agreed, in this case it won't overload the logic too much in the createFungibleTransfer method, and it may be better to directly include it there.
I've refactored it.